cvat-core/src/server-proxy.ts

```diff
@@ -1888,7 +1888,7 @@ async function saveLogs(logs) {
     const { backendAPI } = config;
```

Comment:
Better to rename the function as well. In general we don't send logs now. We send events which can be exported and processed.